src/modules/shared/components/Peak/Peak.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC, useEffect, useRef, useState } from 'react';
+import WaveformData from 'waveform-data';
+
+import { useGetDefaultPeak } from '@shared/hooks/get-default-peak';
+
+import { PeakProps } from './Peak.types';
+
+const Peak: FC<PeakProps> = ({ json }) => {
+	const { data: fallback } = useGetDefaultPeak(!json);
+	const [waveform, setWaveform] = useState<WaveformData | undefined>(undefined);
+
+	const canvas = useRef<HTMLCanvasElement>(null);
+
+	const data = json || fallback;
+
+	useEffect(() => {
+		data && setWaveform(WaveformData.create(data));
+	}, [data]);
+
+	useEffect(() => {
+		const draw = () => {
+			const ctx = canvas.current?.getContext('2d');
+
+			if (!ctx || !waveform) {
+				return;
+			}
+
+			const step = 8;
+			const spacing = step * 0.5;
+
+			const scaleY = (amplitude: number, height: number) => {
+				const range = 128;
+				const offset = 64;
```

Comment:
Hoe beïnvloeden bovenstaande values (`range`, `offset`) de render?

Reply:
🤷, snippet van ergens overgenomen en ik vind de source niet meer terug. De values aanpassen lijkt de verschillen tussen peaks te vergroten.